src/app/map/[id]/Tile/auth.dynamic.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import React, { useState } from "react";
+import { LoginForm } from "~/components/auth/LoginForm";
+import { RegisterForm } from "~/components/auth/RegisterForm";
+
+interface AuthTileProps {
+  initialView?: "login" | "register";
+}
+
+// This component will be dynamically imported
+export default function AuthTile({ initialView = "login" }: AuthTileProps) {
+  const [showLogin, setShowLogin] = useState(initialView === "login");
+
+  return (
```

Comment:
The AuthTile should be wrapped inside a StaticBaseTileLayout (i.e in an Hexagonal shape) for better consistency with the app overall design. While it is not yet displayed inside an HexMap, it will soon be the case.